src/components/documentViewer/documentViewer.js

```diff
@@ -3,17 +3,18 @@ import PropTypes from "prop-types";
 import { documentTemplates } from "./utils";
 
 const DocumentViewer = props => {
-  const { tabIndex, document, handleHeightUpdate } = props;
+  const { tabIndex, document, handleHeightUpdate, obfuscateDocument } = props;
```

Comment:
```suggestion
  const { tabIndex, document, handleHeightUpdate, handleObfuscation } = props;
```

To be more consistent in naming